### PR TITLE
fix-#257: Hide and visible in password to improve UI

### PR DESCRIPTION
**Pull request by GOURAVSINGH19**
## Summary

I improve the sign in and signup page by adding visible and hide password feature which help use to check the password before submit...

https://github.com/krishnaacharyaa/wanderlust/assets/120244528/eb167125-fe64-4de0-8ed5-846aac91aade

### CODE REVIEW

File: frontend/package-lock.json
Comment (krishnaacharyaa) [inline frontend/package-lock.json:1]: Why is there a new package-lock.json?

File: frontend/src/pages/signin-page.tsx [hunk 1]
```diff
@@ -84,15 +88,58 @@ function signin() {
...
               className="w-full rounded-lg bg-zinc-100 p-3 font-normal placeholder:text-sm placeholder:text-neutral-500"
             />
+            <button
+              type="button"
+              className="absolute right-3 top-1/2 -translate-y-1/2 transform focus:outline-none"
+              onClick={togglePasswordVisibility}
+            >
+              {showPassword ? (
+                <svg
+                  xmlns="http://www.w3.org/2000/svg"
```
Comment (krishnaacharyaa) [inline frontend/src/pages/signin-page.tsx:105]: kindly take this svg out and put it in assets and import like the other svgs being used in the project

File: frontend/src/pages/signup-page.tsx [hunk 2]
```diff
@@ -103,17 +107,61 @@ function signup() {
...
-            {errors.password && (
-              <p className="p-3 text-xs text-red-500">{`${errors.password.message}`}</p>
+            <button
+              type="button"
+              className="absolute right-3 top-1/2 -translate-y-1/2 transform focus:outline-none"
+              onClick={togglePasswordVisibility}
+            >
+              {showPassword ? (
+                <svg
+                  xmlns="http://www.w3.org/2000/svg"
```
Comment (krishnaacharyaa) [inline frontend/src/pages/signup-page.tsx:124]: Same comment

File: package-lock.json
Comment (krishnaacharyaa) [inline package-lock.json:1]: Why is there a new package-lock.json?

**Review by krishnaacharyaa — CHANGES_REQUESTED** [review]
Kindly use the standard svg icons from flaticon

File: package-lock.json
Comment (GOURAVSINGH19) [reply package-lock.json:1]: give a time I will fix it

**Comment by krishnaacharyaa** [comment]
Kindly rebase the branch to latest branch, worflow file has some changes, so that is causing this fail in PR title checking job

File: frontend/src/assets/svg/link.svg
Comment (krishnaacharyaa) [inline frontend/src/assets/svg/link.svg:1]: Why have you touched this?

**Comment by GOURAVSINGH19** [comment]
I donot know by it cannot merge..

File: frontend/src/pages/signup-page.tsx [hunk 3]
```diff
@@ -119,46 +121,10 @@ function signup() {
...
-                    strokeLinejoin="round"
-                    strokeWidth={2}
-                    d="M17 21v-2a4 4 0 00-4-4H7a4 4 0 00-4 4v2"
-                  />
-                </svg>
-              )}
+              {showPassword ? <Passwordvisible /> : <Passwordhide />}
             </button>
-            {errors.fullName && (
-              <p className="p-3 text-xs text-red-500">{errors.fullName.message}</p>
```
Comment (krishnaacharyaa) [inline frontend/src/pages/signup-page.tsx]: Why this is changed?

**Review by krishnaacharyaa — CHANGES_REQUESTED** [review]

**Comment by krishnaacharyaa** [comment]
Forget about the PR title check , i'll take care of it.
@GOURAVSINGH19 kindly update what are the icons you have used and don't touch any other files other than what is required

**Comment by krishnaacharyaa** [comment]
@GOURAVSINGH19 kindly address the review comments

File: frontend/src/pages/signup-page.tsx (hunk 3 above)
Comment (GOURAVSINGH19) [reply frontend/src/pages/signup-page.tsx]: I already fix it

**Comment by krishnaacharyaa** [comment]
@GOURAVSINGH19 please change the icons, and don't touch any other icon or update other files, and once all this is done, update the PR description to reflect those icons

**Pull request closed without merging**